tests/affine-migration/e2e/basic.spec.ts

```diff
@@ -38,9 +38,6 @@ test('v1 to v4', async ({ page }) => {
   await expect(page.getByTestId('upgrade-workspace-button')).toBeVisible();
   await page.getByTestId('upgrade-workspace-button').click();
 
-  await expect(page.getByText('Refresh Current Page')).toBeVisible();
```

Comment:
why delete those codes, no upgrade happened?

Reply:
I removed the Refresh button after the update, and now it automatically resumes the normal page after update.